src/main/java/uk/gov/hmcts/reform/profilesync/service/ProfileSyncService.java

```diff
@@ -6,8 +6,6 @@
 import uk.gov.hmcts.reform.profilesync.domain.UserProfileSyncException;
 
 public interface ProfileSyncService {
-    public static String BASIC = "Basic ";
-    public static String BEARER = "Bearer ";
```

Comment:
no need for these to be declared in the interface as they can be declared in the class (implementation)